website/server/libs/routeBlacklist.js

```diff
@@ -0,0 +1,15 @@
+import _ from 'lodash';
+
+const ROUTE_BLACKLIST = [
```

Comment:
Can be removed since now there's an option that can be passed to the auth middleware?